apps/cognito-functions/src/main.ts

```diff
@@ -33,3 +35,22 @@ export const sensEmailHandler = pipe(
       })
   )
 );
+
+export const createAuthChallengeHandler = pipe(
+  { fromEmailAddress: process.env.FROM_EMAIL_ADDRESS },
+  createAuthChallenge.CreateAuthChallengeConfig.decode,
+  E.fold(
+    (errors) => {
+      // eslint-disable-next-line functional/no-expression-statements
+      console.log(PR.failure(errors).join('\n'));
+      // eslint-disable-next-line functional/no-throw-statements
+      throw new Error();
+    },
+    (config) =>
+      createAuthChallenge.makeHandler({
+        config,
+        ses: new SES(),
+        generateVerificationCode: () => crypto.randomBytes(3).toString('hex'),
```

Comment:
I think we should move the `generateVerificationCode` somewhere else and add unit tests to it so we can verify the function will generate OTP in a format we want.
Looking at the tests, we want a 4 digits code, but the `crypto.randomBytes` function returns a 6 chars alphanumeric string.

What do you think?